app/views/posts/_list_item.haml

```diff
@@ -44,8 +44,10 @@
   %td.width-70.post-replies.vtop{class: klass}= link_to replies_count, stats_post_path(post)
   - if logged_in? && local_assigns[:show_unread_count]
     %td.width-70.vtop{class: klass}
-      - if @opened_ids.include?(post.id) && unread_ids.include?(post.id)
-        = Reply.where(post_id: post.id).where('created_at > ?', post.last_read(current_user)).count
+      - if @unread_counts.key?(post.id)
```

Comment:
It seems like this might cause strange behavior if we forget to add `with_unread: true` to `posts_from_relation` but `posts/list_item` expects an unread count. Should we fall back to the n-query version?

Reply:
Do we spec this? if we do it should raise an error and we should... notice

Reply:
Yeah, it errors out if I take off `with_unread: true`, and we feature spec posts#unread and favorites#index. It's -- not _impossible_ something will get added with `show_unread_count` that fails to set that, but I would _expect_ us to check it renders properly :P

Reply:
...it raises an error with `.key?` but not with `.try`, tbc